composer.json

```diff
@@ -6,8 +6,8 @@
     "prefer-stable": true,
     "type": "library",
     "require": {
-        "php": "^8.0",
-        "dvsa/mot-cpms-client": "^1.0.0",
+        "php": "^8.1",
+        "dvsa/mot-cpms-client": "dev-feature-BL-15591-update-cpms-php-81",
```

Comment:
Will branch refs still work once this is merged?

Reply:
Once reviewed, I will create tags and sort it out.